src/net.c

```diff
@@ -26,6 +24,12 @@ struct network {
 };
 
 
+struct laddr {
+	struct le le;
+	struct sa sa;
+};
```

Comment:
I think it is a bit overkill to use a list as a container.
perhaps it is enough with a static array:

```c
struct sa sav[8];
```

how many IP-addresses can a system have ?

Reply:
I started with an array, but found that `list` is simpler.